core/templates/pages/exploration-player-page/services/exploration-recommendations.service.ts

```diff
@@ -33,10 +33,10 @@ import { LearnerExplorationSummary } from
   providedIn: 'root'
 })
 export class ExplorationRecommendationsService {
-  isIframed: boolean = null;
-  isInEditorPage: boolean = null;
-  isInEditorPreviewMode: boolean = null;
-  explorationId: string = null;
+  isIframed: boolean = false;
+  isInEditorPage: boolean = false;
+  isInEditorPreviewMode: boolean = false;
+  explorationId: string | undefined;
```

Comment:
This should not be undefined now, right ?

Reply:
yes, done